src/Forms/DurationsBridgeGaps.cs

```diff
@@ -126,6 +124,7 @@ private void GeneratePreviewReal()
             FixedCount = Core.Forms.DurationsBridgeGaps.BridgeGaps(_fixedSubtitle, minMsBetweenLines, radioButtonDivideEven.Checked, (double)numericUpDownMaxMs.Value, fixedIndexes, _dic);
 
             SubtitleListview1.Fill(_fixedSubtitle);

```

Comment:
`Fill` calles Items.Clear() + BeginUpdate() / EndUpdate()

Reply:
I think this is okay, it clears the list view, and handles a lot of listview stuff (BeginUpdate can be nested I think)

Reply:
> I think this is okay, it clears the list view, and handles a lot of listview stuff (BeginUpdate can be nested I think)

Sure, that is why we don't need remove lines above! I don't know if you noticed, I didn't remove BeginUpdate / EndUpdate.